staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go

```diff
@@ -35,7 +35,7 @@ var (
 		&k8smetrics.HistogramOpts{
 			Name:    "rest_client_request_duration_seconds",
 			Help:    "Request latency in seconds. Broken down by verb, and host.",
-			Buckets: k8smetrics.ExponentialBuckets(0.001, 2, 10),
+			Buckets: []float64{0.005, 0.025, 0.1, 0.25, 0.5, 1.0, 2.0, 4.0, 15.0, 30.0, 60.0},
```

Comment:
there is quite a big gap between 4s and 15s, so I think we should add one additional bucket to improve measurements. I would be fine with either 8s or 10s.

Reply:
donde